### PR TITLE
CIV-5789 -  ccd translation for DQ common screens.

### PR DESCRIPTION
**Pull request by neeta-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-5789

### Change description ###
This PR has code changes for DQ common(SmallClaim/FastTrack Claim) hearing requirement screens.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/services/features/response/submission/submitResponse.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export const submitResponse = async (req: AppRequest): Promise<Claim> => {
     const claimFromCivilService = await civilServiceClient.retrieveClaimDetails(claimId, req);
     const isAddressUpdated = addressHasChange(claim.respondent1?.partyDetails?.primaryAddress, claimFromCivilService?.respondent1?.partyDetails?.primaryAddress);
     const ccdResponse = translateDraftResponseToCCD(claim, isAddressUpdated);
+    console.log('Console: ', ccdResponse);
```
Comment (laperla11) [inline src/main/services/features/response/submission/submitResponse.ts:22]: please remocve the log

File: src/main/services/translation/response/convertToCCDSmallClaimHearing.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      return undefined;
+  }
+}
+
+function toCCDUnavailableDates(dateDetails: UnavailableDatePeriod[]) {
+  if (!dateDetails?.length) return undefined;
+  const smallClaimUnavailableDates = dateDetails.map((unavailableDatePeriod: UnavailableDatePeriod) => {
+    return {
+      value: {
+        who: undefined,
```
Comment (laperla11) [inline src/main/services/translation/response/convertToCCDSmallClaimHearing.ts:24]: isn't it better assign it to a value like `defendant` ?

**Review by laperla11 — APPROVED** [review]

**Review by LeonardoPalmeiro — DISMISSED** [review]

**Review by TatsyMaslova — DISMISSED** [review]

File: src/main/services/features/response/submission/submitResponse.ts (hunk 1 above)
Comment (neeta-hmcts) [reply src/main/services/features/response/submission/submitResponse.ts]: fixed

File: src/main/services/translation/response/convertToCCDSmallClaimHearing.ts (hunk 2 above)
Comment (neeta-hmcts) [reply src/main/services/translation/response/convertToCCDSmallClaimHearing.ts]: fixed.

**Review by laperla11 — DISMISSED** [review]

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by portizdev — APPROVED** [review]

**Review by dtortolaV1 — DISMISSED** [review]

**Review by laperla11 — DISMISSED** [review]

**Review by portizdev — APPROVED** [review]

**Review by laperla11 — APPROVED** [review]

**Review by TatsyMaslova — APPROVED** [review]

**Review by dtortolaV1 — APPROVED** [review]

**Pull request merged**